test/dspal_tester/adsp_proc/serial_test_imp.c

```diff
@@ -56,14 +58,22 @@
  * - make sure to define UART-BAM mappings in /usr/share/data/adsp/blsp.config
  *   See snapdragon flight user guide for details.
  */
-#define MAX_UART_DEVICE_NUM      6
-#define NUM_UART_DEVICE_ENABLED  4
+#define MAX_UART_DEVICE_NUM      1 //6
+#define NUM_UART_DEVICE_ENABLED  1 //4
 
```

Comment:
Either this is "1" and the comment should be removed, or there is a TODO or FIXME not required here